helm-chart/splunk-enterprise/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: splunk-enterprise
+description: A Helm chart for Splunk Enterprise managed by the Splunk Operator
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "2.0.0"
+
+dependencies:
+- name: splunk-operator
+  version: "1.0.0"
```

Comment:
should we change this to 2.0.0 ? , am ok with 1.0.0 also

Reply:
I am ok to have separate chart version than splunk operator.

Reply:
Yes, so it can be anything we want. The version specified above is the chart version for the splunk-operator chart. Currrently on the develop branch it is 1.0.0 for the chart version and 2.0.0 for the appVersion.